src/cli.ts

```diff
@@ -36,9 +43,14 @@ async function main(): Promise<void> {
   let cliConfig: CliConfig;
   try {
     const appJson = (await fse.readJson("./app.json")) as AppJson;
+    const { platforms, ...appJsonConfig } = appJson.svgAppIcon || {};
+
     cliConfig = {
       ...defaultCliConfig,
-      ...appJson.svgAppIcon
+      ...appJsonConfig,
+      ...(Array.isArray(platforms) && {
+        platforms: platforms.map((e) => e.toLowerCase() as Platform)
+      })
     };
```

Comment:
Not sure how in depth the input validation / normalisation should be. Right now every value for `platforms` that isn't an array will be ignored, but arrays containing anything other than strings will cause the CLI to **silently** fall back to default values.

Reply:
I think that is completely fine. It will probably be quite obvious for a user if the platform configuration doesn't work due to invalid input. Similarly to as if there would be a typo on the config field name, or something similar.

If we start encountering more issues with invalid configurations among users, we can start adding more sophisticated validation and error messages. For now that shouldn't be needed.